.github/workflows/kyber.yml

```diff
@@ -0,0 +1,87 @@
+name: Kyber Tests
+
+on:
+  push:
+    branches: [ '*' ]
+  pull_request:
+    branches: [ '*' ]
+
+jobs:
+  build_liboqs:
+    name: Build liboqs
+    runs-on: ubuntu-latest
+    timeout-minutes: 4
+    steps:
+      - name: Install liboqs source
+        run: git clone --depth 1 --branch "0.10.0" "https://github.com/open-quantum-safe/liboqs"
+
+      - name: Build and install liboqs
+        working-directory: liboqs
+        run: |
+          mkdir build
+          cd build
+          cmake -DCMAKE_INSTALL_PREFIX=${{ github.workspace }}/opt -DOQS_MINIMAL_BUILD=KEM_kyber_512 -DOQS_USE_OPENSSL=0 ..
+          make
+          make install
+
+      - name: Upload liboqs
+        uses: actions/upload-artifact@v4
+        with:
+          name: wolfssh-liboqs
+          path: ${{ github.workspace }}/opt/
+          retention-days: 3
+
+  build_wolfssl:
+    name: Build wolfssl
+    runs-on: ubuntu-latest
+    timeout-minutes: 4
+    steps:
+      - name: Install wolfssl source
+        run: git clone --depth 1 "https://github.com/wolfssl/wolfssl"
+
+      - name: Build and install wolfssl
+        working-directory: wolfssl
+        run: |
+          autoreconf -ivf
+          ./configure --prefix=${{ github.workspace }}/opt --enable-wolfssh --enable-cryptonly --disable-examples --disable-crypttests
+          make
+          make install
+
+      - name: Upload wolfssl
+        uses: actions/upload-artifact@v4
+        with:
+          name: wolfssh-wolfssl
+          path: ${{ github.workspace}}/opt/
+          retention-days: 3
```

Comment:
There is no need to split out wolfssl building since its a one (maybe many in the future) to one relationship with wolfssh version.

Reply:
I prefer to split it out, and I'm caching it. I'm testing against a specific version of wolfSSL. I'm using a cache, specifically, instead of an artifact.

Reply:
In that case that is a good choice.